spec/jobs/expire_token_job_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe ExpireTokenJob, type: :job do
+  before { Sidekiq::Testing.inline! }
+
+  subject { described_class.perform_at(token.expired_at, token_id: token.id) }
+
+  let!(:token) { create(:token) }
```

Comment:
Let's apply one conventional call order:
1. Definition of variables -> `let`, `let!`
2. `subject`
3. `before`
4. `after`

Reply:
Agreed, but I vote for `subject`/`let`/`before`/`after`

Reply:
That's the order they use in style guide
https://github.com/reachlocal/rspec-style-guide#good-example-2